Library/Formula/swiftgen.rb

```diff
@@ -4,11 +4,12 @@ class Swiftgen < Formula
   url "https://github.com/AliSoftware/SwiftGen/archive/0.5.0.tar.gz"
   sha256 "555f190f2ffef940eebd80a926eeb05d3d0de573412028c5bd2184e2b9542929"
   head "https://github.com/AliSoftware/SwiftGen.git"
+  revision 1
 
-  bottle do
-    cellar :any
-    sha256 "29391f183e1606e50008ad148b3665d240c28ab5cbd42a293d1ca99b29aa3793" => :el_capitan
-    sha256 "c150775bf2f6b8e77a821eae7bd89233a134eb22ce5eed88de9eaf2822dd79ee" => :yosemite
+  def pour_bottle?
```

Comment:
> Given a not tiny number of compile scripts specifically check for /Applications/Xcode.app and presume you don't have one unless it is named exactly that, having no Xcode.app is quite risky. You can keep multiple Xcodes, but you'll hit significantly fewer issues if one is kept as the default name.

Sure, but that's not always anybody's choice. For example in the case of the OP of https://github.com/AliSoftware/SwiftGen/issues/35#issuecomment-148500711 the issue is on CircleCI, not even his own machine. So I think I should still ensure it works on such contexts.